data/ecosystems/s/scroll.toml

```diff
@@ -11,7 +11,55 @@ sub_ecosystems = [
   "SyncSwap",
   "XY Finance",
   "zkRollups.xyz",
-]
+  "4EVERLAND",
+  "Aave",
+  "Aboard Exchange",
+  "Ankr",
+  "Axelar Network", 
+  "Bware Labs",
+  "BlockPi Network",
+  "Chainge Finance",
+  "Chainlink",
+  "Chainstack",
+  "Coin98",
+  "Deri Protocol",
+  "DODO",
+  "Dune Analytics",
+  "Espresso Systems",
+  "Etherspot",
+  "imToken", 
+  "iZUMi.finance",
+  "Masa Finance",
+  "Mask Network",
+  "Meson",
+  "Metavault DAO",
+  "NerveNetwork",
+  "NFTScan",
+  "OmniBTC",
+  "OpenOcean",
+  "Orbiter Finance",
+  "Poolshark Protocol",
+  "portto (Blocto App)",
+  "Pyth Network",
+  "Redstone Finance",
+  "Reservoir Protocol",
+  "Natural Selection Labs  (RSS3)",
+  "Rubic",
+  "Sablier Finance",
+  "Safe Ecosystem",
+  "Seedify.fund",
+  "sentio.xyz",
+  "Stargate",
+  "Symmetry",
+  "SynFutures",
+  "The Graph",
+  "TokenPocket", 
+  "Transit Finance",
+  "Witnet" ,
+  "Wombat Exchange", 
+  "WOWswap",
+  "Zerion" 
```

Comment:
Overall looks good.  Let's remove Zerion, portto, and Dune Analytics since those are more "General" products.

Reply:
removed